src/main/java/frc/robot/Parameters.java

```diff
@@ -340,15 +346,54 @@
 
     public static final class climber{
             //motor speeds for climbers
-                public static final double CLIMBMOTOR1_SPEED = 0.0;
-                public static final double CLIMBMOTOR2_SPEED = 0.0;
+            //TODO fix names lol
+                public static final double CLIMB_MOTOR_1_SPEED = 0.0;
+                public static final double CLIMB_MOTOR_2_SPEED = 0.0;
+
+                public static final class can{
+                        public static final int CLIMB_MOTOR_1_ID = 13;
+                        public static final int CLIMB_MOTOR_2_ID = 14;
+                }
 
     }
 
     public static final class intake{
             //motor speed for intake
-            public static final double INTAKEMOTORBOTTOM_SPEED = 0.0;
-            public static final double INTAKEMOTORTOP_SPEED = 0.0;
+            public static final double INTAKE_MOTOR_BOTTOM_SPEED = 0.0;
+            public static final double INTAKE_MOTOR_TOP_SPEED = 0.0;
+
+            public static final class can{
```

Comment:
A subclass for can isn't necessary here, it just increases verbosity.

Reply:
I agree. The can class in the drivetrain was only because we had 8 motors